packages/lesswrong/lib/alignment-forum/posts/custom_fields.ts

```diff
@@ -93,6 +93,7 @@ addFieldsDict(Posts, {
     insertableBy: ['members', 'sunshineRegiment', 'admins'],
     editableBy: ['members', 'alignmentForum', 'alignmentForumAdmins'],
     optional: true,
+    hidden: true,
```

Comment:
@Discordius Curious about the best way to make this field changeable by member accounts without being viewable to them.